lib/Cake/Console/Command/TestShell.php

```diff
@@ -226,6 +226,8 @@ protected function _runnerOptions() {
 
 		if (!empty($params['no-colors'])) {
 			unset($params['no-colors'], $params['colors']);
+		} elseif (strtolower(substr(php_uname(), 0, 3)) === 'win') {
```

Comment:
There are a number of windows consoles that _do_ support escape codes from what I remeber. ANSIcon being one.